README.md

```diff
@@ -78,7 +78,7 @@ is handled automatically
 by [klt-cert-manager](https://github.com/keptn/lifecycle-toolkit/blob/main/klt-cert-manager/README.md).
 Versions 0.6.0
 and earlier have a hard dependency on the [cert-manager](https://cert-manager.io).
-See [installation guideline](https://github.com/keptn/lifecycle-toolkit/blob/main/docs/content/en/docs/snippets/tasks/install.md)
+See [installation guideline](https://lifecycle.keptn.sh/docs/install/cert-manager/)
```

Comment:
This has been removed from the README.md in https://github.com/keptn/lifecycle-toolkit/pull/1862 -- not sure if you want to remove this change from your PR or deal with it later.